src/frontend/src/handler/alter_barrier.rs

```diff
@@ -0,0 +1,152 @@
+// Copyright 2025 RisingWave Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use pgwire::pg_response::{PgResponse, StatementType};
+use risingwave_common::catalog::DatabaseParam;
+use risingwave_sqlparser::ast::ObjectName;
+
+use super::{HandlerArgs, RwPgResponse};
+use crate::Binder;
+use crate::catalog::DatabaseId;
+use crate::error::{ErrorCode, Result};
+use crate::session::SessionImpl;
+
+pub async fn handle_set_barrier_interval(
```

Comment:
Can merge the two `handle_set_xxx` to a single `handle_set_database_param`, and pass `AlterDatabaseParam`.